### ISSUE TITLE
npm install @neaps/tide-prediction doesn't not install

### ISSUE THREAD

**Issue by gabrielsalvador** [neaps/tide-predictor#152]
npm install @neaps/tide-prediction does not work,
I'm guessing the name has been updated to tide-predictor

so it should be installed using npm install @neaps/tide-prediction

**Comment by kevee** [comment]
That is correct, fixed in #151 and will release in NPM soon. Will close this issue once the new version is released on NPM.
